.gitignore

```diff
@@ -0,0 +1,17 @@
+TestData2.csv 
+
+FaultyDataTest2.csv  
+FaultyDataTest.csv
```

Comment:
This is interesting, it was a good idea to test some faulty data on your script. What types of faulty data did you test, and what were the results?

Reply:
The try statement catches any exceptions to handling the CSV files. I think this could be useful in the event that someone accidentally made a typo entering data. 

Consider a character value that can't be converted to a floating point number, or a mislabeled column name. I tested both and both were caught. The errors were too vague though in my opinion, so I added a try statement to the function handling conversion of cash value strings to floats that helped (which isn't included in this PR)... I have to admit this feels like somewhat rudimentary and clumsy error handling to me but I think it's a useful consideration nonetheless. 

The script in this PR always logs the CSVs that contain the sums, btw - which I remediated in a subsequent PR. 

I made a couple of edits to improve the error handling slightly and created a new PR, if you care to check them out. Otherwise, I'm excited for round 3!